src/Publisher.php

```diff
@@ -89,7 +89,7 @@ public function generatePageResponse($url)
         }
         try {
             $ssl = Environment::getEnv('SS_STATIC_FORCE_SSL');
-            if (is_null($ssl)) {
+            if (!$ssl) {
```

Comment:
Will this work with all the variations like 'on' 'off' 'true' 'false' '1' '0'?

Wasn't there some utility that was created to normalise reading environment variables?